src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs

```diff
@@ -17,46 +17,65 @@ namespace Microsoft.Health.Fhir.TemplateManagement.Client
     public class OrasClient : IOrasClient
     {
         private readonly string _imageReference;
+        private readonly Regex _digestRx = new Regex("(?<rule>[A-Za-z][A-Za-z0-9]*([+.-_][A-Za-z][A-Za-z0-9]*)*):(?<digest>[0-9a-fA-F]{32,})");
```

Comment:
Can we have a better naming instead of "Rx"?

Reply:
We can name *rule* and *digest* to *algorithm* and *hex*. The total part is called digest

Reply:
Thanks, fixed it.